.github/ISSUE_TEMPLATE/new-diag-study-template.md

```diff
@@ -0,0 +1,77 @@
+---
+name: New Diagnostic Template
+about: Systematically summarize each therapy-focused paper
+title: 'New Paper (Diagnostics): [Title]'
+labels: New Paper
+assignees: ''
+
+---
+
+<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->
+
+Title: Please edit the title to add the name of the paper after the colon
+
+### Please paste a link to the paper or a citation here:
+
+### What is the paper's [DOI](https://library.uic.edu/help/article/1966/what-is-a-doi-and-how-do-i-use-them-in-citations)?
+
+
+### Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19
+
+
+<!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer them now, please go ahead and interleave your answers with the prompts. If you are not ready, feel free to go ahead and create the issue now. -->
+
+### Questions to answer about each paper:
+
+* **1-2 sentences introducing study, main findings**
+* **Study question(s) being investigated:** 
+	* Is the study focused on diagnostics or treatments?
+	If it is on therapeutics, use new-therapy-study-template instead.
+	* What type of testing scenario is being considered?
+	Is it a screening test (used for individuals with no symptoms), diagnostic test (used for individuals with symptoms), or definitive test (used for individuals who have had previous positive test results on diagnostic or screening tests)?
+	* What reference test is considered as a "gold standard" comparator for the test under investigation?
```

Comment:
Is this different that the reference test item below? I wasn't sure, so I removed this. When my PR comes in you can see. I wanted to note it here so that if I misunderstood, we could correct my mistake :)